lib/perl/centreon/script/centcore.pm

```diff
@@ -568,7 +591,7 @@ sub sendExportFile($){
 
     # Send data with SCP
     $self->{logger}->writeLogInfo("Start: Send export files on poller $id");
-    $cmd = "$self->{scp} -r -P $port $origin $dest 2>&1";
+    $cmd = "$self->{scp} -P $port $origin $dest 2>&1";
```

Comment:
the YAML files are in the second level in export this means we need recursively copy too

Reply:
updating